src/Core/Domain/OrderReturn/OrderReturnSettings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\OrderReturn;
+
+class OrderReturnSettings
+{
+    public const ORDER_RETURN_STATE_WAITING_FOR_PACKAGE_ID = 2;
```

Comment:
Maybe you should complete this with the other states even if they are all integers and even if they are not used yet in the recent handler But at least it will allow identifying the different steps/states

Ideally you could also replace the hard-coded values in the legacy code with these consts

Reply:
Oh wait.. I just realized that this is actually an ID from the DB based on the fixtures... And we even use some checks like `if ($orderReturn->state < OrderReturnSettings::ORDER_RETURN_STATE_WAITING_FOR_PACKAGE_ID)` So basically our domain rules are based on the assumption that the order states were added in a specific order and never modified since Even though they are editable 🤦‍♂️

Reply:
I know it's not your fault and this is all from the legacy, but I really don't think we should continue handling this feature this way. We need to see with @MatShir how the return states are supposed to behave and improve them while we are doing this migration

Reply:
Moved PDF and order return logic to separate PR
https://github.com/PrestaShop/PrestaShop/pull/32048